server/src/rest/routes/refunds.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { wrapResponse, requireAuth } from '../middleware';
+
+export const refunds = Router();
+refunds.use(requireAuth());
+
+refunds.post('/initiate', wrapResponse(
```

Comment:
Technically, it can be added to users, or even transactions. The reason I decided to create a new route for this is to remain consistent with `donations` which also have their own route. But one could make the sense they could be be under users, or even top-level routes.

As for the second question, you cannot make a refund request without being logged-in. Notice the `refunds.use(requireAuth())` at the top, it applies to all endpoints in the `refunds` router.

Reply:
@habbes Oh cool, I hadn't notice `refunds.use(requireAuth())`